src/app/profile/yourGiving/recipientView/recipientGift/recipientGift.tpl.html

```diff
@@ -34,10 +34,10 @@ <h4 class="mb0">{{$ctrl.recipient['designation-name']}}</h4>
             </div>
           </div>
           <div class="col-sm-4 col-md-4">
-            <div class="form-group mb0" ng-if="$ctrl.recipient['mostrecentdonationRecurringdonationelement']">
+            <div class="form-group mb0" ng-if="$ctrl.recipient['recurringdonations']['donations'].length">
               <label translate>Recurring Gifts:</label>
               <ul class="list-unstyled list-recurring">
-                <li ng-repeat="recurring in $ctrl.recipient['mostrecentdonationRecurringdonationelement']">
+                <li ng-repeat="recurring in $ctrl.recipient['recurringdonations']['donations']">
```

Comment:
No, the recurringdonations element is a single link with an array of donations now.